docs/maintainers/sass-compilation.md

```diff
@@ -0,0 +1,47 @@
+# play-frontend-hmrc: SASS compilation
+
+This library manages SASS compilation for you. However, should you wish to add your own for any reason using 
+`sbt-sassify`, please read the steps below.
+
+1.  Import the required styles in your `app/assets/stylesheets/application.scss` file:
```

Comment:
I would recommend doing this instead to bring in the mixins only and continue to use hmrcHead:

```
@import "lib/govuk-frontend/govuk/base";
```